src/pybind/mgr/dashboard/module.py

```diff
@@ -517,6 +517,12 @@ class Root(object):
             def default(self, *args, **kwargs):
                 if module.get_module_option('standby_behaviour', 'redirect') == 'redirect':
                     active_uri = module.get_active_uri()
+
+                    if cherrypy.request.path_info == '/api/prometheus_receiver':
```

Comment:
Just to cope with things like `/api/prometheus_receiver/` or `/api/prometheus_receiver?foo=bar` 
```suggestion
                    if cherrypy.request.path_info.startswith('/api/prometheus_receiver'):
```